DEVELOPER.md

```diff
@@ -274,7 +261,8 @@ You can create a _distribution_ (like what you can download from Apache) as foll
 do not use the Maven release plugin because creating an official release is the task of our release manager.
 
     # First, build the code.
-    $ mvn clean install  # you may skip tests with `-DskipTests=true` to save time
+    $ mvn clean install -Pstorm-core # you may skip tests with `-DskipTests=true` to save time
+    $ mvn clean install -Pstorm-more # you may skip tests with `-DskipTests=true` to save time
```

Comment:
No, its profiles are different. It shows that build steps need to be split into two.